src/library/components/resource-lightbox.js

```diff
@@ -242,20 +242,32 @@ var ResourceLightbox = Component({
     }
 
     var license = resource.license_info;
+    var license_attribution = '';
+    // don't provide suggested attribution for public domain resources
+    if (license.code !== 'CC-BY 4.0') {
+      license_attribution = p({}, 'Suggested attribution: ' + resource.name + ' by ',
+              a({href: 'https://concord.org/'}, 'The Concord Consortium'),
+              ' is licensed under ',
+              a({href: license.deed}, license.code),
+              '.'
+            );
+    }
```

Comment:
What about resources that CC doesn't own? We have a few of these in the portal. So if they choose anything other than CC-BY 4.0 we are going to suggest that concord should be attributed.

I suspect we are going to need to move this into the data model somehow.

Reply:
I suggest pulling this change out of this PR so we can get the rest of the changes merged in.

Reply:
The portal sends a 'credits' field that can be used for this. If it isn't set then we could set it to Concord Consortium.